duden/word.py

```diff
@@ -344,3 +344,14 @@ def words_before(self):
     def words_after(self):
         """Returns 5 words after this one in duden database"""
         return [name for name, _ in self.before_after_structure['Im Alphabet danach']]
+
+    @property
+    def phonetic(self):
+        """
+        Returns pronunciation of the word in phonetic notation (https://en.wikipedia.org/wiki/International_Phonetic_Alphabet)
```

Comment:
Could you please wrap this line to maximum 99 characters? `flake8` checker complains:
```shell
$ make check
autopep8 --max-line-length 99  --diff -r duden/ tests/ | colordiff
flake8 --builtins="_" --max-line-length 99 duden/ tests/
duden/word.py:351:100: E501 line too long (126 > 99 characters)
make: *** [Makefile:11: check] Error 1
```